pkg/maps/nodemap/cell.go

```diff
@@ -14,10 +15,24 @@ var Cell = cell.Module(
 	"eBPF map which contains information about node IDs and their IP addresses",
 
 	cell.Provide(newNodeMap),
+	cell.Config(defaultConfig),
 )
 
-func newNodeMap(lifecycle cell.Lifecycle) bpf.MapOut[Map] {
-	nodeMap := newMap(MapName)
+type Config struct {
+	NodeMapMax uint32 `mapstructure:"node-map-max"`
+}
+
+func (c Config) Flags(fs *pflag.FlagSet) {
+	fs.Uint32("node-map-max", defaultConfig.NodeMapMax,
```

Comment:
It looks like all other sibling flags are prepended with `bpf-` (cf. https://docs.cilium.io/en/latest/network/ebpf/maps/#ebpf-maps). Do we want to do the same here?